docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.diagnostics.md

```diff
@@ -1,7 +1,12 @@
+---
+title: Office.context.mailbox.diagnostics
```

Comment:
Should these Outlook OM titles reflect the version?

Reply:
I initially omitted this info b/c CATS restricts titles to a certain max length (for SEO reasons, I believe). However, I agree that this is important info from the customer perspective, so I've added it...we can revisit if/when we see CATS errors for these titles being too long.